views/layout.erb

```diff
@@ -63,58 +63,8 @@
         <a href="https://apps.lib.umich.edu/my-account/favorites">My Favorites</a>
       </div>
     </m-website-header>
-    <form class="search-box" role="search" method="post" action="<%= ENV.fetch('BASE_URL') %>/search">
-      <div class="viewport-container search-form">
-        <div class="search-box-dropdown">
-          <select aria-label="Search or browse by" autocomplete="off" name="type">
-            <% fields.each do |group| %>
-              <optgroup label="<%=group[:label]%>">
-                <% group[:options].each do |option| %>
-                  <option <%=option[:disabled]%> <%=option[:selected]%> value="<%=option[:value]%>">
-                    <%=option[:label]%>
-                  </option>
-                <% end %>
-              </optgroup>
-            <% end %>
-          </select>
-          <m-icon name="keyboard-arrow-down"></m-icon>
-        </div>
-        <input aria-label="Browse text" <% if list.error? %>aria-invalid="true" aria-describedby="input-message"<% end %> type="text" autocomplete="on" name="query" value="<%= list.original_reference %>">
-        <% if list.error? %>
-          <span class="message message--error" id="input-message" role="alert">
-            <%= list.error_message %>
-          </span>
-        <% end %>
-        <button>
-          <m-icon name="search"></m-icon>
-          <span class="visually-hidden">Search</span>
-        </button>
-      </div>
-      <div class="viewport-container search-tip">
-        <m-icon name="info-outline"></m-icon>
-        <% fields.each do |group| %>
-          <% group[:options].each do |option| %>
-            <p data-option="<%=option[:value]%>" style="display: <%= option[:selected] ? 'initial' : 'none' %>;">
-              <span class="strong"><%=group[:tip]%>:</span>
-              <%=option[:tip]%>
-            </p>
-          <% end %>
-        <% end %>
-      </div>
-    </form>
-    <div class="datastores-nav">
-      <nav class="viewport-container" aria-label="Datastores">
-        <ol>
-          <% datastores.each do |datastore| %>
-            <li>
-              <a href="<%= datastore.href %>" <% if datastore.current? %>aria-current="page"<% end %>>
-                <%= datastore.label %>
-              </a>
-            </li>
-          <% end %>
-        </ol>
-      </nav>
-    </div>
+    <%#= erb :'layout/search_box' %>
```

Comment:
@niquerio I need your help with this. It takes in `list`, but is seen as undefined and throws an error.